tools/scaffold/template/model.tmpl

```diff
@@ -8,18 +8,18 @@ import (
 )
 
 // TODO: `ctx` parameter and `diags` return value can be removed if tf schema has no complex data types (e.g., schema.ListAttribute, schema.SetAttribute)
-func NewTF{{.NamePascalCase}}(ctx context.Context, apiResp *admin.{{.NamePascalCase}}) (*TF{{.NamePascalCase}}Model, diag.Diagnostics) {
+func NewTFModel(ctx context.Context, apiResp *admin.{{.NamePascalCase}}) (*TFModel, diag.Diagnostics) {
```

Comment:
[q] do we currently generate models for nested objects?

Reply:
the current tool generates only the main schema with evertything inside together. We have to manually extract the schemas we want to functions.